src/Nethermind/Nethermind.Merkleization/Merkle.BitArray.cs

```diff
@@ -8,17 +8,17 @@ namespace Nethermind.Merkleization;
 
 public static partial class Merkle
 {
-    public static void IzeBitvector(out UInt256 root, BitArray value)
+    public static void Ize(out UInt256 root, BitArray value)
```

Comment:
I suggest renaming this cryptic `Ize()` method to a more understandable one, say, `Merklize` or `Merkleize`. Perhaps the former. Whatever we choose, we need to have a consistent naming.

Reply:
Took `Merkleize` from the spec